system-contracts/contracts/EvmInterpreterFunctions.template.yul

```diff
@@ -801,13 +760,8 @@ function _pushEVMFrame(_passGas, _isStatic) {
 
 function _popEVMFrame() {
     // function popEVMFrame() external
-    // 0xe467d2f0
-    let selector := 0xe467d2f0
 
-    mstore8(0, 0xe4)
-    mstore8(1, 0x67)
-    mstore8(2, 0xd2)
-    mstore8(3, 0xf0)
+    mstore(0, 0xE467D2F000000000000000000000000000000000000000000000000000000000)
```

Comment:
@jrchatruc Is there a possibility that this will override some bytes that are important before the call?